libraries/LiquidCrystal/src/LiquidCrystal.h

```diff
@@ -44,22 +44,22 @@
 
 class LiquidCrystal : public Print {
 public:
-  LiquidCrystal(uint8_t rs, uint8_t enable,
-		uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3,
-		uint8_t d4, uint8_t d5, uint8_t d6, uint8_t d7);
-  LiquidCrystal(uint8_t rs, uint8_t rw, uint8_t enable,
-		uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3,
-		uint8_t d4, uint8_t d5, uint8_t d6, uint8_t d7);
-  LiquidCrystal(uint8_t rs, uint8_t rw, uint8_t enable,
-		uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3);
-  LiquidCrystal(uint8_t rs, uint8_t enable,
-		uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3);
-
-  void init(uint8_t fourbitmode, uint8_t rs, uint8_t rw, uint8_t enable,
-	    uint8_t d0, uint8_t d1, uint8_t d2, uint8_t d3,
-	    uint8_t d4, uint8_t d5, uint8_t d6, uint8_t d7);
+  LiquidCrystal(const uint8_t rs, const uint8_t enable,
+		const uint8_t d0, const uint8_t d1, const uint8_t d2, const uint8_t d3,
+		const uint8_t d4, const uint8_t d5, const uint8_t d6, const uint8_t d7);
```

Comment:
Making code clear is great, but I think this overuse of `const` in places where it essentially does nothing for the user is hindering the readability. This will only ever benefit someone writing the implementation (if even then).